LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/font-size-adjust-012-expected.txt

```diff
@@ -0,0 +1,48 @@
+layer at (0,0) size 800x600
```

Comment:
Tests that have a rendertree dump as reference can't be exported to WPT.

You need another HTML file as reference, or the test has to be using the JS testharness.